src/supports.js

```diff
@@ -13,3 +14,19 @@ export const removeEventListener = canUseDOM && 'removeEventListener' in window;
 // IE8+ Support
 export const attachEvent = canUseDOM && 'attachEvent' in window;
 export const detachEvent = canUseDOM && 'detachEvent' in window;
+
+// Passive options
+// Inspired by https://github.com/Modernizr/Modernizr/blob/master/feature-detects/dom/passiveeventlisteners.js
+export const passiveOption = () => {
```

Comment:
I was returning a function accidentally... 😢 
How do you think is the immediate function that returns a boolean?

```javascript
export const passiveOption = (() => {
  let supportsPassiveOption = false;
  // ...
  return supportsPassiveOption;
})();
```